app/src/main/java/com/futurice/freesound/feature/common/ui/adapter/RecyclerViewAdapter.java

```diff
@@ -38,10 +38,10 @@
 /**
  * Implementation of {@link RecyclerView.Adapter} for {@link DisplayableItem}.
  */
-public final class RecyclerViewAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+public final class RecyclerViewAdapter<T> extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
```

Comment:
The purpose of this PR is not to remove `DisplayableItem` yet. It was to fix it a bit as omitting the generic part was making it type unsafe.
In the future, we could think about how this could be done better in statically typed way